notebooks/WHISP_and_chains_the_easy_way.ipynb

```diff
@@ -110,7 +185,7 @@
         "credentials, _ = google.auth.default(\n",
         "    scopes=['https://www.googleapis.com/auth/earthengine']\n",
         ")\n",
-        "ee.Initialize(credentials, project='forest-data-partnership', opt_url='https://earthengine-highvolume.googleapis.com')\n",
+        "ee.Initialize(credentials, project=PROJECT, opt_url='https://earthengine-highvolume.googleapis.com')\n",
```

Comment:
this will just write PROJECT into the file, not the value it holds. That's fine but will lead to errors on turn-up for the cloud function. Consider adding a comment instructing the user to replace PROJECT? Otherwise you can write a custom magic function that supports parameters and get a the desired result, e.g. with https://github.com/nopdotcom/jupyter-jinja-yaml-magic or [this hack](https://github.com/ipython/ipython/issues/6701#issuecomment-382640776). Note that you probably need the surrounding quotes too.